### DISCUSSION TITLE
How exactly can it be achieved?

### DISCUSSION THREAD

**Discussion by sinalalebakhsh** [community/community#21055] [Discussions] [answered]
please anwser me :
step by step simple way.
to Zero IQ 👍

**Comment by jfullstackdev** [comment, marked as answer]
what is that you want to achieve ? there are so many badges from GitHub, specify those you want

**Reply by sinalalebakhsh** [reply]
for famus

**Reply by Iwezuble055** [reply]
How can i licence?

**Reply by yusuftemel** [reply]
You're able to get the Pull Shark Achievement by opening pull requests that eventually are merged with a repo. You can get the YOLO badge if you merge a pull request without a review. 
After answering two discussions, You will get Galaxy Brain badge
More details here:
https://github.com/Schweinepriester/github-profile-achievements
Be wise with this one, hahaha! Best of luck!

**Comment by Gametrek** [comment]
Your IQ is the ability to retain knowledge as fast possible. IQ tests are not designed for people who have learned knowledge ( like the SAT ). It is design to find out how fast you can figure things out. Take that and apply to academics is what most people uses IQ for. However the same could be said for survival, procreation, and other feats. Gangues Khan was known to be the worlds most successful biological father but he fell off a horse that trampled him. Similar Edgar Allan Poe is known for the creation of the American language as we know it today but yet he died by falling into a puddle of water getting sick of pneumonia.

**Reply by sinalalebakhsh** [reply]
OK 
My question not very series .
Any way thank you

**Comment by thuongtruong1009** [comment]
Hi @sinalalebakhsh 
Well, it's great that you like all the achievement badges. I'm just like you, so I'm always working to find and improve my achievements. Here are the badges that I earned.
![image](https://user-images.githubusercontent.com/71834167/180589203-c554fdec-a4d8-48a2-a5ea-6fc922c67ac8.png)
See at [my profile](https://github.com/thuongtruong1009)
Thank you!

**Reply by sinalalebakhsh** [reply]
Hello back 
🙏🏼💐
How your earned?

**Reply by thuongtruong1009** [reply]
> Hello back 🙏🏼💐 How your earned?

This is a way to earn. See at [achievement](https://github.com/Schweinepriester/github-profile-achievements)
Please mark this comment as an answer to help more people and practice together
Thank you!

**Reply by sinalalebakhsh** [reply]
thanks
